hw/dv/tools/testplans/tl_device_access_types_testplan.hjson

```diff
@@ -12,7 +12,21 @@
     {
       name: illegal_access
       desc: '''Drive unsupported requests via TL interface and verify correctness of response
-            / behavior '''
+            / behavior. Below error cases are tested
+            - TL-UL protocol error cases
+              1. Unsupported opcode. e.g a_opcode isn't Get, PutPartialData or PutFullData
+              2. Mask isn't all active if opcode = PutFullData
+              3. Mask isn't in enabled lanes, e.g. a_address = 0x00, a_size = 0, a_mask = 'b0010
+              4. Mask doesn't align with address, e.g. a_address = 0x01, a_mask = 'b0001
+              5. Address and size aren't aligned, e.g. a_address = 0x01, a_size != 0
+              6. Size is over 2.
+            - OpenTitan defined error cases
+              1. Access unmapped address, return d_error = 1 when devmode_i == 1
+              2. Write CSR with unaligned address, e.g. a_address[1:0] != 0
+              3. Write CSR less than its width, e.g. when CSR is 2 bytes wide, only write 1 byte
+              4. Write a memory without enabling all lanes (a_mask = '1) if memory doesn't support
+                byte enabled write
+              5. Read a WO (write-only) memory'''
       milestone: V2
       tests: ["{name}_tl_errors"]
     }
```

Comment:
Also, I think the`{intf}` in the test names below can be removed. Also if possibe, expand more on the partial_access description.

Reply:
Updated.